content/master/concepts/managed-resources.md

```diff
@@ -839,96 +839,139 @@ Providers may define their own custom `Conditions`.
 
 
-### Available
-`Reason: Available` indicates the Provider created the managed resource and it's
-ready for use. 
+### Type: Ready
 
-```yaml {copy-lines="none"}
+Condition `Type: Ready` indicates if the external resource is ready to use.
+
+The Condition `Type: Ready` and `Status: True` indicates Provider created
+the external resource and notified Crossplane it's ready to use.
+
+{{<hint "important" >}}
+Crossplane doesn't update `Status: True` during a resource update.
+{{< /hint >}}
+
+The Condition `Type: Ready` and `Status: False` indicates the external resource
+isn't available. 
+
+
+#### Available
+{{<hover label="available" line="4">}}Reason: Available{{</hover>}} indicates 
+the Provider created the managed resource and it's ready for use. 
+
+```yaml {copy-lines="none",label="available"}
 Conditions:
   Type:                  Ready
   Status:                True
   Reason:                Available
 ```
-### Creating
 
-`Reason: Creating` indicates the Provider is attempting to create the managed
-resource. 
+#### Creating
 
-```yaml {copy-lines="none"}
+{{<hover label="creating" line="4">}}Reason: Creating{{</hover>}} indicates the 
+Provider is attempting to create the managed resource. 
+
+```yaml {copy-lines="none",label="creating"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Creating
 ```
 
-### Deleting
-`Reason: Deleting` indicates the Provider is attempting to delete the managed
-resource. 
+#### Deleting
+{{<hover label="deleting" line="4">}}Reason: Deleting{{</hover>}} indicates the 
+Provider is attempting to delete the managed resource. 
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="deleting"}
 Conditions:
   Type:                  Ready
   Status:                False
   Reason:                Deleting
 ```
 
-<!-- vale off -->
-### ReconcilePaused
-<!-- vale on -->
-`Reason: ReconcilePaused` indicates the managed resource has a [Pause](#paused)
-annotation 
+#### Unavailable
+{{<hover label="unavailable" line="4">}}Reason: Unavailable{{</hover>}}
+indicates Crossplane expects the managed resource to be available, but the 
+Provider reports the resource is unhealthy.
 
-```yaml {copy-lines="none"}
+```yaml {copy-lines="none",label="unavailable"}
 Conditions:
-  Type:                  Synced
+  Type:                  Ready
   Status:                False
-  Reason:                ReconcilePaused
+  Reason:                Unavailable
 ```
 
+### Type: Synced
+
+Condition `Type: Synced` indicates Crossplane has checked the status of the managed
+resource with the Provider.
+
+The condition `Type: Synced` and `Status: True` means Crossplane successfully
+communicated with the Provider on the status of the manged resource. 
+
+{{< hint "note" >}}
+`Type: Synced` and `Status: True` only shows success between Crossplane and 
+the Provider. This status doesn't mean the Provider successfully changed the
+external resource.
+{{< /hint >}}
```

Comment:
This isn't what the synced condition represents. 🙂 

The synced condition represents whether a resource's controller made it through the most recent reconcile without hitting an error.

This this context, this means the synced condition represents whether the provider successfully synced the MR's desired state (spec) with the external system, and successfully synced the observed state of the external system back to the MR (status).

Reply:
Synced means the same for claims and XRs, at their level of abstraction.

So a claim is synced when its controller successfully syncs the claim's desired state with an XR. An XR is synced when its controller successfully syncs its desired state with its composed resources.